packages/office-ui-fabric-react/src/components/FloatingPicker/Suggestions/SuggestionsControl.tsx

```diff
@@ -30,15 +30,15 @@ export class SuggestionsHeaderFooterItem extends BaseComponent<ISuggestionsHeade
   public render(): JSX.Element {
     const { renderItem, onExecute, isSelected, id } = this.props;
     return onExecute ? (
-      <button
+      <div
```

Comment:
I don't know the context of this, but having a div that has a click target will not be focusable. I also am curious if a role here is missing, as this likely wouldn't be read by screen readers.